ci-operator/step-registry/rosa/cluster/upgrade/hcp/rosa-cluster-upgrade-hcp-ref.yaml

```diff
@@ -0,0 +1,34 @@
+ref:
+  as: rosa-cluster-upgrade-hcp
+  from: rosa-aws-cli
+  grace_period: 10m
+  commands: rosa-cluster-upgrade-hcp-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 300Mi
+  timeout: 15h0m0s
```

Comment:
Can prow support 15 hours running for the job?

Reply:
Updated it to a valid timeout for this step